gdal/ogr/ogrsf_frmts/flatgeobuf/ogr_flatgeobuf.h

```diff
@@ -144,19 +154,43 @@ class OGRFlatGeobufLayer final : public OGRLayer
         void VerifyBuffers( int bFlag ) { m_bVerifyBuffers = CPL_TO_BOOL(bFlag); }
 
         const std::string& GetFilename() const { return m_osFilename; }
+
+        static std::string GetTempFilePath(CPLString fileName, char **papszOptions);
+
+        uint16_t GetIndexNodeSize() const { return m_indexNodeSize; } 
+        OGRwkbGeometryType GetOGRGType() const { return m_eGType; } 
```

Comment:
you'd perhaps want to override GetGeomType() instead (but only if GetGeomType() always equals to GetLayerDefn()->GetGeomType())

Reply:
I find it hard to determine if I can guarantee that GetGeomType() always equals to GetLayerDefn()->GetGeomType())